### PR TITLE
[12.0] mig account_due_list

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: account_due_list/__manifest__.py [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 {
     'name': 'Payments Due list',
-    'version': '11.0.2.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Generic Modules/Payment',
+    'development_status': 'Beta',
```
Comment (pedrobaeza) [inline account_due_list/__manifest__.py:7]: I think this deserve a `Mature` status

**Review by pedrobaeza — COMMENTED** [review]
The rest seems OK

File: account_due_list/__manifest__.py (hunk 1 above)
Comment (lmignon) [reply account_due_list/__manifest__.py:7]: @pedrobaeza This addons doesn't conform to all criteria defined for 'Mature' https://docs.google.com/document/d/1MmGxv0yTw0pvB-PYAcUhbCcku9ZG-Y2DSGXwZDNYyIw/edit#heading=h.i0zrunr17t42 (no tests, no declared maintainer, ...)

Comment (pedrobaeza) [reply account_due_list/__manifest__.py:7]: @lmignon OK. I have seen all in green and thought that there was tests. About the maintainer, I didn't know that requirement.

Comment (eLBati) [reply account_due_list/__manifest__.py:7]: Indeed, I put Beta because tests are missing

**Review by mrcast — APPROVED** [review]
I tested due list for invoice from vendors and customers, I tested filters

**Comment by eLBati** [comment]
Mergeable?

**Pull request merged**